client/gutenberg/extensions/slideshow/style.scss

```diff
@@ -2,6 +2,17 @@
 
 	margin-bottom: 1.5em;
 	position: relative;
+	.wp-block-jetpack-slideshow_swiper-wrappper {
+		padding: 0;
+		margin: 0;
+		line-height: normal;
+	}
+
+	// Override UL left margin from theme styles
+	.edit-post-visual-editor & ul.wp-block-jetpack-slideshow_swiper-wrappper {
+		margin: 0;
+	}
```

Comment:
It might be possible to absorb this selector into the one above (L5-L9):

```scss
	// Need the extra specificity to override UL left margin from theme styles
	.edit-post-visual-editor & .wp-block-jetpack-slideshow_swiper-wrappper {
		padding: 0;
		margin: 0;
		line-height: normal;
	}
```

Not sure which is uglier :smile:

Reply:
Yeah, I separated it out so it would be visually obvious what the brute force override was all about. I'd be happy to clean up though, if you think that's better.

Reply:
I don't have a strong preference. Makes sense to single out the brute force override I guess.